### ISSUE TITLE
Documentation:  secrets.kms_key_id

### ISSUE THREAD

**Issue by mowings** [terraform-aws-modules/terraform-aws-rds-proxy#14]
The example shows this as a guid. It needs to be an arn, else the creation of the role will fail.

**Comment by vishnu-techprescient-teika** [comment]
I am also facing the same issue. Attaching error log for reference :

```terraform
module.http_test_db.module.proxy[0].aws_iam_role_policy.this[0]: Creating...
╷
│ Error: putting IAM role policy example-proxy: MalformedPolicyDocument: Resource <kms_key_id> must be in ARN format or "*".
│       status code: 400, request id: <request_id>
│ 
│   with module.http_test_db.module.proxy[0].aws_iam_role_policy.this[0],
│   on .terraform/modules/http_test_db.proxy/main.tf line 173, in resource "aws_iam_role_policy" "this":
│  173: resource "aws_iam_role_policy" "this" {
│ 
```

**Comment by jriveralivaque** [comment]
i have same problem, please your support

**Comment by antonbabenko** [comment]
This issue has been resolved in version 3.0.0 :tada:
